rmt-helm/README.md

```diff
@@ -6,7 +6,7 @@ It is tested on K3s but should work on any Kubernetes distribution.
 ## Overview
 
 To deploy SUSE RMT on top of Kubernetes, each component of the stack is deployed in a dedicated container via a
-Helm Chart.
+Helm chart available on the [helm-charts GitHub repository](https://github.com/SUSE/helm-charts/tree/main/rmt-helm).
```

Comment:
I think this is unnecessary ( the helm chart should not refer its own source location)